Cargo.toml

```diff
@@ -58,27 +57,58 @@ unit_bindings = "warn"
 unused_import_braces = "warn"
 unused_lifetimes = "warn"
 unused_macro_rules = "warn"
-# unused_qualifications = "warn" # Try to re-enable when MSRV is > 1.76
+unused_qualifications = "warn"
 
 [lints.clippy]
 cargo = { level = "warn", priority = -1 }
 nursery = { level = "warn", priority = -1 }
 pedantic = { level = "warn", priority = -1 }
-multiple_crate_versions = "allow"
+allow_attributes = "warn"
+allow_attributes_without_reason = "warn"
```

Comment:
Ah, these all default to "allow", but we don't want that.  Did you go through and add all of these manually or is there a list somewhere that we're able to use?

Reply:
Manual. There may be others we want to add.